benchmarks/report.R

```diff
@@ -65,6 +65,8 @@ GetOshLabel = function(shell_hash, prov_dir) {
       label = 'osh-ovm'
     } else if (length(grep('bin/osh', lines)) > 0) {
       label = 'osh-cpython'
+    } else if (length(grep('_bin/.*/mycpp-souffle/osh', lines)) > 0) {
```

Comment:
OK yeah I see here is the heuristic I was thinking of

I think there is some existing smell where we should clarify the relationship between shell output and R output

it is a bit fuzzy now